components/MembershipDriveRoadmap/DevelopmentDomain.js

```diff
@@ -0,0 +1,106 @@
+      img_phase2: "/assets/images/Group_web_phase2.png",
+      phaseDetails: {
+        phase1:
+          "HTML , CSS,  JAVASCRIPT (DOM+ES6 VERSION), DOM and Version Control System like Git and Github.\nProject to grasp all the concepts and apply it. ",
+        phase2:
+          "Get started with Object Oriented Programming(OOP) in Js , React Js and Redux.",
+        phase3:
+          "Get started with Firebase services.\nMake a fully functional webapp implementing all the concepts learnt in the previous phases.",
+      },
+    },
+    {
+      devDomainName: "App Dev",
+      id: 2,
+      className: style.app_dev_card,
+      logo: "/assets/images/Android.webp",
+
+      description:
+        "Android is the most widely used mobile OS with more than 70% of market share.\nKotlin and Java are the primary language to build android applications",
+      img_phase2: "/assets/images/Android.webp",
+      phaseDetails: {
+        phase1:
+          "Get started with Java and OOP (Object oriented programming).\nLearn VCS (Version control system) like Git and GitHub. Make CLI programs.",
+        phase2:
+          "Get started with Android Development.\nLearn Android architecture and working with Android Studio.\nMake a simple app.",
+        phase3:
+          "Get started with Firebase services.\nMake a fully functional app implementing all the concepts learnt in the previous phases.",
+      },
+    },
+    {
+      devDomainName: "Flutter",
+      id: 3,
+      className: style.flutter_dev_card,
+      logo: "/assets/images/Flutter_logo.png",
+      description:
+        "Flutter is a Portable UI framework by Google. You can build for Android, iOS, Web and desktop apps with it.\nDart is the language you will use in Flutter.",
+      img_phase2: "/assets/images/Group_Flutter_phase2.png",
+      phaseDetails: {
+        phase1:
+          "Get started with Dart and Java.\nLearn VCS (Version control system) like Git and GitHub.\nMake CLI programs",
+        phase2:
+          "How Flutter works and some core concepts of Native Dev (Android).\nMake a simple Android and Flutter app.",
+        phase3:
+          "Get started with Firebase services.\nMake a fully functional app implementing all the concepts learnt in the previous phases.",
+      },
+    },
+  ];
+
+  return (
+    <div className="container-fluid">
+      <div className="row">
+        <div className={`container-fluid ${style.devDomainChoiceTitle}`}>
+          <img
+            className={style.choose_tool_img}
+            alt="choose_tool"
+            src="/assets/images/choose_tool_ninja.png"
+          />
+          <div> Choose your Tool</div>
+        </div>
+        <div className={`container-fluid ${styles.btnGroupWrap}`}>
+          <div className="btn-group flex-wrap mt-5" role="group">
+            {domainButtons.map((domain) => {
```

Comment:
Make a component for the button and all the attributes as props instead of doing this

Reply:
See TeamCard Component of homepage for reference

Reply:
The TeamCard component has directly implemented these buttons with repetition of code for which a better way is using the `map()` method.

Reply:
Where is the repetition of of code in that component?

Reply:
Making a component would be more modular and cleaner

Reply:
My bad in the Index the component is used multiple times instead `map()` could be used and as for the buttons I'll make a component that you'll have call in [index.js](https://github.com/TSEC-MAD-Club/MAD-Website/blob/bde8235a024d4b608fde6c4eaecac3706c887d87/pages/index.js#L116)